tests/unit/wdl/test_preseq_lc_extrap_defects.wdl

```diff
@@ -0,0 +1,20 @@
+version 1.0
+
+
+import "../../../wdl/tasks/preseq.wdl"
+
+
+workflow test_preseq_lc_extrap_defects {
```

Comment:
Hmm. I don't think you need both this and `unit/wdl/test_preseq_lc_extrap_no_defects.wdl`. Can reuse the WDL, just change the JSON you are feeding it.